sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1213,11 +1213,16 @@ SIREPO.app.directive('modalEditor', function(appState, panelState) {
                 $scope.modalTitle = viewInfo.title;
             }
             $scope.$on('modelChanged', function (e, name) {
-                if (name == $scope.modelKey) {
+                if (name == $scope.modelName) {
```

Comment:
This change breaks places which have multiple copies of the same model (same modelName but unique modelKeys). For example, in elegant on the source panel, select a report, change a value and select "Save Changes". The modal dialog doesn't close in that case. The same problem exists in SRW on the beamline watchpoint reports.

Reply:
4a52fad

I can't remember why I changed it or even that I changed it!